File: Serene/Serene.Script/Imports/FormContexts/Meeting.MeetingLocationForm.cs

namespace Serene.Meeting
{
    using Serenity;
    using Serenity.ComponentModel;
    using System;
    using System.Collections;
    using System.Collections.Generic;
    using System.ComponentModel;
    using System.Runtime.CompilerServices;
    using Serene.Northwind;

    public partial class MeetingLocationForm : PrefixedContext
    {
        public MeetingLocationForm(string idPrefix) : base(idPrefix) {}
    
        public StringEditor Name { get { return ById<StringEditor>("Name"); } }
        public StringEditor IsActive { get { return ById<StringEditor>("IsActive"); } }
        public StringEditor Address { get { return ById<StringEditor>("Address"); } }
    }
}

File: Serene/Serene.AspNetCore/Imports/ServerTypings/Meeting.MeetingLocationForm.ts
namespace Serene.Meeting {
    export class MeetingLocationForm extends Serenity.PrefixedContext {
        static formKey = 'Meeting.MeetingLocation';

    }

    export interface MeetingLocationForm {
        Name: Serenity.StringEditor;
        Address: Serenity.StringEditor;
        Latitude: Serenity.DecimalEditor;
        Longitude: Serenity.DecimalEditor;
    }

    [['Name', () => Serenity.StringEditor], ['Address', () => Serenity.StringEditor], ['Latitude', () => Serenity.DecimalEditor], ['Longitude', () => Serenity.DecimalEditor]].forEach(x => Object.defineProperty(MeetingLocationForm.prototype, <string>x[0], { get: function () { return this.w(x[0], (x[1] as any)()); }, enumerable: true, configurable: true }));
}

